devserver/src/components/Playground.tsx

```diff
@@ -16,7 +17,6 @@ import type { SideContentTab } from "./sideContent/types";
 import { getDynamicTabs } from "./sideContent/utils";
 
 // Importing this straight from js-slang doesn't work for whatever reason
```

Comment:
Can we move this comment too? (I know the reordering was automatic due to lint rules)